app/templates/main/index.html

```diff
@@ -4,24 +4,131 @@
+                <label>Option</label>
+              </div>
+              <div class="ui checkbox field one wide column">
+                <input type="checkbox" name="option">
+                <label>Option</label>
+              </div>
+              <div class="ui checkbox field one wide column">
+                <input type="checkbox" name="option">
+                <label>Option</label>
+              </div>
+              <div class="ui checkbox field one wide column">
+                <input type="checkbox" name="option">
+                <label>Option</label>
+              </div>
+              <div class="ui checkbox field one wide column">
+                <input type="checkbox" name="option">
+                <label>Option</label>
+              </div>
+              <div class="ui checkbox field one wide column">
+                <input type="checkbox" name="option">
+                <label>Option</label>
+              </div>
+              <div class="ui checkbox field one wide column">
+                <input type="checkbox" name="option">
+                <label>Option</label>
+              </div>
+            </div>
+          </div>
+
+          <div id="advanced-search" class="option-group-advanced">
+            <div class="descriptor-group">
+              <h5>DESCRIPTOR #2</h5>
+              <div class="ui checkbox field one wide column">
+                <input type="checkbox" name="option">
+                <label>Option</label>
+              </div>
+              <div class="ui checkbox field one wide column">
+                <input type="checkbox" name="option">
+                <label>Option</label>
+              </div>
+              <div class="ui checkbox field one wide column">
+                <input type="checkbox" name="option">
+                <label>Option</label>
+              </div>
+            </div>
+
+            <div class="descriptor-group">
+              <h5>DESCRIPTOR #3</h5>
+              <div class="ui checkbox field one wide column">
+                <input type="checkbox" name="option">
+                <label>Option</label>
+              </div>
+              <div class="ui checkbox field one wide column">
+                <input type="checkbox" name="option">
+                <label>Option</label>
+              </div>
+              <div class="ui checkbox field one wide column">
+                <input type="checkbox" name="option">
+                <label>Option</label>
+              </div>
+            </div>
+          </div>
+
+          <div id="advanced-filters-showhide">
+              Show Advanced Filters
+          </div>
         </div>
+
+        <div class="ui divider"></div>
+        <div id="list" class="ui middle aligned divided list"></div>
       </div>
+
       <div class="twelve wide column">
         <div id="map"></div>
       </div>
     </div>
 
+    <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.1.1/jquery.min.js">
```

Comment:
usually we want to leave script src imports at the top (in this case right after googlemapsapi import at the top of this file) and custom javascript at the bottom